src/Reference/HasOne_SQL.php

```diff
@@ -45,8 +48,12 @@ public function addField($field, $their_field = null) : Field_SQL_Expression
             $their_field = $field;
         }
 
+        // if caption is not defined in $defaults -> get it directly from the linked model field $their_field
+        $defaults['caption'] = $defaults['caption'] ?? $this->owner->refModel($this->link)->getField($their_field)->getCaption();
```

Comment:
this is problematic when there is more than 1 reference, when the meaning of imported field is too broad - like `name` - in `user` table it is clear what is means, but if imported into `order` table, we `name` has completely different meaning

@abbadon1334 what is your motivation behind this?

would you be ok if we drop this and the test you added?